sbt_common/internal_model/src/main/scala/uk/ac/wellcome/models/work/internal/Work.scala

```diff
@@ -27,13 +32,15 @@ trait Work extends MultipleSourceIdentifiers {
   val visible: Boolean
 
   val ontologyType: String
+  val redirect: Option[Redirect]
```

Comment:
Instead of putting redirect on work could we create a `RedirectedWork` that shared a sealed trait with `Work`?

Reply:
That would allow us to enforce that visible == false if redirect is not-None.

Reply:
Yep, I think we could extend that idea for non visible works as well and avoid the horrible error when a work is visible but has no title